Shared/Samples/Browse OGC API feature service/BrowseOGCAPIFeatureServiceView.swift

```diff
@@ -0,0 +1,260 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct BrowseOGCAPIFeatureServiceView: View {
+    /// The tracking status for the loading operation.
+    @State private var isLoading = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// Ensures that the alert is only shown once.
+    @State private var presentAlert = true
+    
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The user input for the OGC service resource.
+    @State private var userInput = "https://demo.ldproxy.net/daraa"
+    
+    /// The selected layer name.
+    @State private var selection: String = ""
+    
+    var body: some View {
+        MapViewReader { mapProxy in
+            MapView(map: model.map)
+                .onDrawStatusChanged { drawStatus in
+                    // Updates the state when the map's draw status changes.
+                    withAnimation {
+                        if drawStatus == .completed {
+                            isLoading = false
+                        }
+                    }
+                }
+                .overlay(alignment: .center) {
+                    if isLoading {
+                        ProgressView("Loading…")
+                            .padding()
+                            .background(.ultraThickMaterial)
+                            .cornerRadius(10)
+                            .shadow(radius: 50)
+                    }
+                }
```

Comment:
For most services we don't want the loading progress view - that gives the user a impression that our API isn't performant 🥴 . Unless it takes more than 5 seconds (which makes people think the app hangs), I tend not to use a progress view.